app/controllers/application_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   include Pundit::Authorization
+  include ActiveStorage::SetCurrent
```

Comment:
This shouldn't be necessary, unless you generate the service url directly (e.g. `user.image.url`) but in general [it's recommended](https://edgeguides.rubyonrails.org/active_storage_overview.html#serving-files) that you use the route helpers to generate an activestorage path (e.g. `url_for(user.image)` or `rails_blob_path(user.image)`)

for that reason I think I'd leave this change out of the base. Thoughts?

Reply:
I'm not sure actually. What if the developer wants/needs to use `user.image.url`. Also, isn't `url_for` a view helper only?

Reply:
<img width="819" alt="Captura de pantalla 2023-08-04 a la(s) 12 34 41" src="https://github.com/rootstrap/rails_api_base/assets/27789496/5c20b1ec-2618-4677-9cda-8f5d40ac8023">